lib/iris/tests/unit/fileformats/pp_load_rules/test__epoch_date_hours.py

```diff
@@ -24,30 +21,30 @@
 #
 
 
-class TestEpochHours__standard(tests.IrisTest):
-    def setUp(self):
+class TestEpochHours__standard:
+    def setup_method(self):
```

Comment:
@pp-mo Same again, convert to fixture pattern? (et al below)

Reply:
as above